ngraph/core/reference/include/ngraph/runtime/reference/experimental_detectron_topk_rois.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cmath>
+#include <cstddef>
+#include <cstdint>
+#include <ngraph/runtime/host_tensor.hpp>
+#include <vector>
+#include "ngraph/node.hpp"
+#include "ngraph/op/util/op_types.hpp"
+#include "ngraph/ops.hpp"
+#include "ngraph/shape_util.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            template <typename T>
+            void experimental_detectron_topk_rois(const T* input_rois,
+                                                  const T* input_probs,
+                                                  const Shape& input_rois_shape,
+                                                  const Shape& input_probs_shape,
+                                                  size_t max_rois,
+                                                  T* output_rois)
+            {
+                const int64_t input_rois_num = static_cast<int64_t>(input_rois_shape[0]);
```

Comment:
Why do you use `int64_t` if all input arguments have a type `size_t`?

Reply:
Fixed.